x-pack/plugins/cloud_security_posture/server/routes/setup_routes.ts

```diff
@@ -46,6 +55,34 @@ export function setupRoutes({
   defineBulkActionCspBenchmarkRulesRoute(router);
   defineGetCspBenchmarkRulesStatesRoute(router);
 
+  core.getStartServices().then(async ([coreStart]) => {
+    const soClient = coreStart.savedObjects.createInternalRepository();
+    await migrateCdrDataViews(soClient, logger);
+  });
```

Comment:
Following up on the previous comment, if we have this function being checked on the `getDataViewSafe` we might not need to have it on the setup.